web/src/main/java/nl/tehdreamteam/se42/web/ServiceContainer.java

```diff
@@ -0,0 +1,41 @@
+package nl.tehdreamteam.se42.web;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Keeps track of all {@link Service Services} and provides utility methods to execute on all {@code Services} at once.
+ */
+public final class ServiceContainer {
+
+    private final Collection<Service> services = new ArrayList<>();
+
+    /**
+     * Starts all registered {@link Service Services}.
+     */
+    public void startAll() {
+        services.forEach(Service::start);
+    }
+
+    /**
+     * Stops all registered {@link Service Services}.
+     */
+    public void stopAll() {
+        services.forEach(Service::stop);
+    }
+
+    /**
+     * Registers a {@link Service} to this {@code ServiceContainer}.
+     */
+    public void addService(Service service) {
+        services.add(service);
+    }
+
+    /**
+     * Deregisters a {@link Service} from this {@code ServiceContainer}.
+     */
+    public void removeService(Service service) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document the parameter(s): service [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)